README.md

```diff
@@ -97,17 +97,17 @@ or multiple files matching `.semgrep/**/*.yml`.
 
 * **Metavariables like `$X`, `$WIDGET`, or `$USERS_2`.** Metavariable names can
 only contain uppercase characters, or `_`, or digits, and must start with
-an uppercase character or `_` - names like `$x` or `$some_value` are
+an uppercase character or `_`. Names like `$x` or `$some_value` are
 invalid.  Metavariables are used to track a variable across a specific code
 scope.
 * **The `...` (ellipsis) operator.** The ellipsis operator abstracts away
-sequences so you don't have to sweat the details of a particular code pattern.
+sequences of zero or more arguments, statements, or characters.
```

Comment:
Does "arguments, statements, or characters" cover all places one can use the ellipsis operator? Not sure if [binary operations](https://github.com/returntocorp/semgrep/blob/develop/docs/pattern-features.md#binary-operations) or [arrays](https://github.com/returntocorp/semgrep/blob/develop/docs/pattern-features.md#arrays) falls under those. Not a huge deal since it gets the point across, but I was being a bit intentionally vague since you can use the ellipsis operators in quite a few places.

Reply:
Great point @mschwager  -- I updated to reflect and link into the deeper docs!